### PR TITLE
Fix all the typos throughout the codebase

### PR DESCRIPTION
**Pull request by im-mortal**
discord com_channels_373745291289034763_498934055564476416

Perhaps a little more than just typos.

### CODE REVIEW

File: locale/English_en.ftl [hunk 1]
```diff
@@ -506,14 +506,14 @@ tt_diagnostics_trigger_diagnostics_on_table_edit_tip = Enable this to trigger a
...
 version = Version:
-header_len = Header Lenght:
+header_len = Header Length:
 codec_four_cc = Codec Four CC:
 width = Width:
 height = Height:
 ms_per_frame = Ms Per Frame:
 num_frames = Number of Frames:
 largest_frame = Largest Frame:
-mistery_number = I don't know what this is:
```
[the remark this answers is not in the document]
Comment (Frodo45127) [reply locale/English_en.ftl:516]: That thing might be a remnant of when I got CA_VP8 support in. There are quite a few unused translations like that.

File: locale/English_en.ftl [hunk 2]
```diff
@@ -638,7 +642,7 @@ nested_table_accept = Accept
 
 about_check_template_updates = Check Template Updates
 uodate_templates_success = Templates updated correctly.
-tt_uodate_templates = This command attemps to update your templates.
```
[the remark this answers is not in the document]
Comment (Frodo45127) [reply locale/English_en.ftl:641]: yeah, same as with the mistery number, no longer used.

File: rpfm_ui/ui_templates/tool_unit_editor.ui [hunk 3]
```diff
@@ -1045,7 +1045,7 @@
            <item row="1" column="0">
             <widget class="QToolButton" name="variantmesh_check_button">
              <property name="text">
-              <string>...</string>
+              <string>…</string>
```
[the remark this answers is not in the document]
Comment (Frodo45127) [reply rpfm_ui/ui_templates/tool_unit_editor.ui:1048]: I really don't know how to type that elipsis symbol quickly, that's why I use the dots. It's ok either way really.

[the remark this answers is not in the document]
Comment (im-mortal) [reply rpfm_ui/ui_templates/tool_unit_editor.ui:1048]: I'm on macOS, it makes life easier.
On Windows 10 or later these symbols can be accessed via Emoji Picker by pressing <kbd>Win</kbd> + <kbd>.</kbd>.

**Review by Frodo45127 — COMMENTED** [review]
Ok everything revised, and testing it locally seems to work. The only thing I'm not really fond of is that hr that's misaligned.

File: locale/English_en.ftl [hunk 4]
```diff
@@ -700,38 +704,47 @@ diagnostic_show = Show?
...
+            <ul><li>Only those two fields of all tables in that folder will be ignored.</li></ul>
         </li>
         <li>
-        <i><b>db/land_units_tables/table1;;DiagId1,DiagId2</b></i> => Only those two diagnostics for that specific table will be ignored. Filter keys available in the manual.
+            <code>db/land_units_tables/table1;;DiagId1,DiagId2</code>
+            <ul><li>Only those two diagnostics for that specific table will be ignored. Filter keys available in the manual.</li></ul>
         </li>
     </ul>
-    <br/>
+    <hr>
```
[the remark this answers is not in the document]
Comment (im-mortal) [reply locale/English_en.ftl:735]: I copied [Russian_ru.ftl](../../../im-mortal/rpfm/blob/fix/language/locale/Russian_ru.ftl#L2085) html part of the translation string there. I didn’t experiment with text strings and QT Creator enough, so I can't think of a better way to improve UI around these elements using translation strings alone.

I believe this kind of decoration doesn't really belong to the Fluent part of the program, so I'll revert both instances of `<hr>`, then. Any further improvements to the UI should be done via layout changes made with QT Creator, which, if ever happens, will be in a separate PR.

**Comment by Frodo45127** [comment]
Ok, with the hr changed, I don't have any more issues with the pr. Merging.

**Comment by im-mortal** [comment]
Thank you! Happy Halloween! <sub>(if you celebrate it)</sub>

**Pull request merged**